lib/src/server/server.dart

```diff
@@ -87,6 +87,7 @@ class ConnectionServer {
   final Map<String, Service> _services = {};
   final List<Interceptor> _interceptors;
   final CodecRegistry? _codecRegistry;
+  final Function? _errorHandler;
```

Comment:
This should be tighter typed (here and other places which accept it), e.g. 

```
typedef ErrorHandler = void Function(GrpcError error, StackTrace? trace);
```
otherwise users don't actually know what kind of function they are expected to pass in.

Reply:
✅